src/firebase/saga.test.ts

```diff
@@ -0,0 +1,37 @@
+import { initializeFirebase, waitForFirebaseAuth } from 'src/firebase/saga'
+import { expectSaga } from 'redux-saga-test-plan'
+import { call } from 'redux-saga/effects'
+import { getAccount } from 'src/web3/saga'
+import { initializeAuth, initializeCloudMessaging } from 'src/firebase/firebase'
+import firebase from '@react-native-firebase/app'
+
+jest.mock('@react-native-firebase/app', () => ({
+  app: () => ({ options: { databaseURL: 'https://test-firebase.valoraapp.com' } }),
+}))
+jest.mock('src/firebase/firebase')
+jest.mock('@celo/utils/lib/async')
+
+describe('firebase saga', () => {
```

Comment:
these tests don't actually test anything new anymore; I wrote them when I was doing this a different way. But we have them now, so I left them in.